packages/xstate-wallet/src/store/channel-store-entry.ts

```diff
@@ -255,7 +255,7 @@ export class ChannelStoreEntry {
     }
 
     const {signedStates} = this;
-    const turnNums = _.map(signedStates, s => s.turnNum.toHexString());
+    const turnNums = _.map(signedStates, s => parseInt(s.turnNum.toHexString(), 16));
```

Comment:
:nit: Does this work with turn numbers bigger than 53 bits?

Perhaps, while we're still using `BigNumber` for turn numbers (I think we should _stop_ doing that), we should change `isReverseSorted` to accept `BigNumber[]`, and work correctly.

Reply:
It seems to work with `ethers.constants.MaxUint256`:
```node
> parseInt(MaxUint256.toHexString(), 16) < parseInt(MaxUint256.sub(1).toHexString(), 16)
false
```